ioservice/cob_foms.c

```diff
@@ -723,12 +725,19 @@ static int cob_ops_fom_tick(struct m0_fom *fom)
 				m0_fom_phase_set(fom, M0_FOPH_TXN_COMMIT_WAIT);
 				return M0_FSO_AGAIN;
 			}
+
+			if (cob_op->fco_fop_type == M0_COB_OP_CREATE &&
+			    m0_be_tx_state(tx) < M0_BTS_LOGGED) {
```

Comment:
We want to "sync" for all object creation operations?
No matter if the "M0_OOF_SYNC" flag is set or not?

Actually, this "M0_OOF_SYNC" is not passed from client to server for cob creation/deletion?

Reply:
Currently there is no option of passing this flag from client side, which can be taken as improvement
or may be covered as part of DTM design.
I think for cob deletes we may not need to wait for sync operation to be completed.
Do you see any issue with it ?

Reply:
OK